ipn/ipnlocal/local.go

```diff
@@ -5327,6 +5333,17 @@ func peerCanProxyDNS(p tailcfg.NodeView) bool {
 	return false
 }
 
+func (b *LocalBackend) SetSleep(enabled bool) {
+	b.logf("SetSleep: enabled = %v", enabled)
+	b.cc.SetSleepMode(enabled)
+	b.MagicConn().SetHomeless(enabled)
+	if enabled {
+		logtail.Disable()
+	} else {
+		logtail.Enable()
+	}
```

Comment:
This doesn't next. This newly added "Enable" means somebody who really wanted it off (e.g. a Headscale user who got it disabled via the control plane) now gets it re-enabled here when their phone screen turns on or off.

Reply:
Good catch!